internal/settings/settings.go

```diff
@@ -36,13 +37,13 @@ import (
 
 const (
 	APIURLEnvVar              = "SPLUNK_API_URL"
-	BallastEnvVar             = "SPLUNK_BALLAST_SIZE_MIB"
```

Comment:
Do we want to let people know to change this env var to GOMEMLIMIT?

Reply:
My idea was GOMEMLIMIT env var do not have effect in the binary  or in other orders you cannot anything once the process start. The idea of declaring it here is just to check if the env is set already ( via helm chart or not ).  I think letting people know about this via Helm chart ( where it is effective ) should be more useful. We can update customer facing document for that.